apis/r/src/arrow.cpp

```diff
@@ -99,7 +99,9 @@ void createSchemaFromArrow(const std::string& uri,
     tiledb::VFS vfs(*ctxsp);
     //spdl::warn("[createSchemaFromArrow] About to create {}, dir exists ? {}", uri, vfs.is_dir(uri));
 
-    if (!vfs.is_dir(uri)) {
+    if (vfs.is_dir(uri)) {
```

Comment:
I'm curious how this works with `tiledb://...` URIs -- ?

Reply:
It should if you have a env vars set etc.  I test more frequently with plain s3, the rest on top "is just us".

```r
> tiledb::tiledb_vfs_is_dir("s3://tiledb-dirk/penguins")
[1] TRUE
> tiledb::tiledb_vfs_is_dir("s3://tiledb-dirk/does_not_exist")
[1] FALSE
> 
```

Reply:
@eddelbuettel I'm referring to `tiledb://...` URIs not `s3://...` URIs

Reply:
The `$exists` method is storage-independent -- it works on more than only VFS.

Reply:
@eddelbuettel thank you.

```
$ rg bool.exists
src/soma/soma_dataframe.h
99:    static bool exists(std::string_view uri);

src/soma/soma_dense_ndarray.h
99:    static bool exists(std::string_view uri);

src/soma/soma_sparse_ndarray.h
99:    static bool exists(std::string_view uri);
```

`exists` is exposed to users in Python and R; also it's in `libtiledbsoma`.

`vfs` likewise exists in Python, R, and C++.

Regardless of language, `vfs` addresses local disk and S3 but does not handle `tiledb://...` URIs.

This PR is now merged, so we'll need to address this non-functioning with respect to `tiledb://...` URIs in a separate PR.

(Note as always that we don't unit-test `tiledb://...` URIs within this repo as this is an open-source repo.)

Reply:
https://github.com/single-cell-data/TileDB-SOMA/issues/2714